index.rst

```diff
@@ -1,61 +1,546 @@
+    all command-response pairs and all technical data streams including
+    hardware health, and status information. The telemetry shall include all
+    required information (metadata) needed for the scientific analysis of the
+    survey data as well as, at a minimum, the following: Changes in the
+    internal state of the system, Health and status of operating systems, and
+    Temperature, rate, pressure, loads, status, and conditions at all sensed
+    system components.
+
+It is a rater generic and broad requirement that specifies that components must
+publish operational status information.
+
+LSE-62
+^^^^^^
+
+The LSST Observatory Control System Requirements Document :cite:`LSE-62`
+contains three requirements regarding system configuration:
+
+Requirement OCS-REQ-0045 in section 3.4.4 (Subsystem Latest Configuration)
+states that:
+
+        Specification: The Configuration Database shall manage the latest
+        configuration for each subsystem, for the different observing modes.
+
+        Discussion: The Configuration Database maintains also the latest
+        configuration utilized during operations that can be utilized for rapid
+        restoration of service in case of failure.
+
+
+Requirement OCS-REQ-0069 in section 3.4.4.1 (Subsystem Parameters) state that:
+
+    Specification: The Configuration Database shall manage the subsystem
+    parameters for the different observing modes.
+
+
+Requirement OCS-REQ-0070 in section 3.4.4.2 (Subsystem History) state that:
+
+    Specification: The Configuration Database shall manage subsystem history
+    for the different observing modes.
+
+LSE-150
+^^^^^^^
+
+Section 2.4 of the LSST Control Software Architecture :cite:`LSE-150` describes
+how to perform configuration management. The document provides two valid
+alternatives for managing configuration in the LSST system; through a
+configuration database or version control system.
+
+For configuration database any solution is acceptable as long as the technology
+allow versioning of the database.
+
+For version control systems the adopted solution is
+`git <https://git-scm.com>`__. The document also specify that configuration
+must be stored in a separate repository, to allow the configuration to evolve
+independently of the main code base.
+
+Interface definition
+--------------------
+
+Assuming the architecture specified in LSE-150 :cite:`LSE-150` for the use of
+configuration database or version controlled configuration repository we must
+now make sure enough information is captured to satisfy the traceability
+requirements specified in LSE-62 :cite:`LSE-62`.
+
+LSE-209 :cite:`LSE-209` specify the global interface a component must implement
+to be a valid resource of the system, including a set of events to satisfy the
+aforementioned configuration requirements, e.g.; ``settingVersions``,
+``settingsApplied`` and ``softwareVersions`` events. At the time of this writing,
```

Comment:
I concur with Michael here.

Reply:
So, I was not planning on mentioning the `softwareVersions` event at all in this document, it just slept over here.. I can if you think it is useful but it seems a little bit out of topic for the purpose of this document.

Reply:
While I agree it doesn't perfectly fit with the rest of the document, it still fits in terms of reporting configurations - as the software version is a type of configuration.  And an explanation needs to go somewhere, there is no where else it can really go, so here is actually the best place.  I think a quick paragraph here will outline what is expected from the CSCs and the developers will know what to do regarding this event.  Thanks Tiago.